### PR TITLE
Adding ServerConfiguration.ts and configurations.json in server folder

### PR DESCRIPTION
**Pull request by molhamkhalil**

### CODE REVIEW

File: server/Configurations.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+	"hostName": "localhost:8080",
+	"port": "8080",
+	"dataLocalPath": "/home/mkhalil/versioned/cogneco/exclusive/build/data",
+	"appPath": "/home/mkhalil/versioned/cogneco/exclusive/build",
+	"authorisationServer": "imint.highrisehq.com",
+	"authorisationPath": "/me.xml"
+}
```
Comment (fredrikbryntesson) [inline server/Configurations.json]: This should not be committed. And you probably do not need full path as dataLocalPath. From project root should be enough.

File: server/Configurations.json~ [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+	"hostName": "localhost:8080",
+	"port": "8080",
+	"dataLocalPath": "./versioned/cogneco/exclusive/build/data",
+	"appPath": "./versioned/cogneco/exclusive/build",
+	"authorisationServer": "imint.highrisehq.com",
+	"authorisationPath": "/me.xml"
+}
```
Comment (fredrikbryntesson) [inline server/Configurations.json~]: This is a backupfile created by your text editor, it should never be committed.

**Comment by molhamkhalil** [comment]
okay I will handle it.

File: server/Configurations.json~ (hunk 2 above)
Comment (molhamkhalil) [inline server/Configurations.json~]: okay I will handle it

File: server/source/Service.ts [hunk 3]
```diff
@@ -69,7 +69,7 @@ module Exclusive {
 							if (!newUser)
 								connection.Write("Bad Request", 400, { 'Content-Type': 'text/html; charset=utf-8' });
 							else {
-								newUser.Create(Exclusive.HostName, this.users, (created: boolean) => {
+								newUser.Create(ServerConfiguration.HostName + ":" + ServerConfiguration.Port.toString() , this.users, (created: boolean) => {
 									if (!created)
```
Comment (fredrikbryntesson) [inline server/source/Service.ts:72]: Space between function call and comma.

**Comment by molhamkhalil** [comment]
Configurations.json not committed

**Comment by molhamkhalil** [comment]
Configurations.json not commited.

**Comment by jonathanudd** [comment]
Looks good to me. @fredrikbryntesson thanks for giving feedback while I was gone. Do you have any more input?

**Pull request merged**